### DISCUSSION TITLE
Remotes как вытянуть из цикла данные по номеру заказа, если номером заказа не является ID цикла

### DISCUSSION THREAD

**Discussion by Qb1cc** [totumonline/totum-ru-issues-and-discussions#159] [Как это сделать?] [answered]
Сейчас в цикл нырнуть можно только по строке ID , но у меня номера заказов привязаны не к ID , как починякать? 
Грубо говоря пользователь будет на странице вписывать номер своего заказа, и ему должны выводится данные по позициям в заказе. 

код Remotes
`=: select(table: 's6_in_cycles'; cycle: $#post[id];  sfield: 'h_h_pokupatel'; )`

![image](https://user-images.githubusercontent.com/84081305/173854253-21669289-817e-45b8-aaa6-ed4c65e9f729.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Надо через `select` найти номер `id` строки в таблице циклов по вашим условиям.

```
=: select(table: 'table_in_cycle'; cycle: $select; field: 'fueld')

select: select(table: 'cycles'; field: 'id'; where: 'number' = $#input)
```

**Reply by Qb1cc** [reply]
Когда не хватает образования, и не знаешь как построить правильно логику, возникают подобные вопросы , но когда видишь на них ответ, кажется всё очень очевидным))
